xwiki-commons-core/xwiki-commons-job/src/main/java/org/xwiki/job/AbstractJob.java

```diff
@@ -204,7 +204,7 @@ protected void runInContext()
             jobStarting();
 
             runInternal();
-        } catch (Throwable t) {
+        } catch (Exception t) {
```

Comment:
This is a false positive. We catch Throwable on purpose since we don't control the Job and we want to be sure to store what crashed it in the log.

Reply:
Suggestion: add a comment to explain that.